File: 백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.cc
#include <stdio.h>

int main(void) {
	int n;
	char name[30];
	scanf("%d", &n);
	for (int i = 0; i < n; i++) {
		scanf("%s", name);
		switch (name[0]) {
		case 'A':
			if (name[1] == 'l') printf("204\n");
			else printf("302\n");
			break;
		case 'D':
			printf("207\n");
			break;
		case 'C':
			printf("B101\n");
			break;
		case 'N':
			printf("303\n");
			break;
		case 'S':
			printf("501\n");
			break;
		case 'T':
			printf("105\n");
			break;
		}
	}
	return 0;
}

File: 백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        for(int i = 0; i < n; i++) {
            String name = br.readLine();
            switch(name.charAt(0)) {
                case 'A':
                    if (name.charAt(1) == 'l') System.out.println("204");
                    else System.out.println("302");
                    break;
                case 'D':
                    System.out.println("207");
                    break;
                case 'C':
                    System.out.println("B101");
                    break;
                case 'N':
                    System.out.println("303");
                    break;
                case 'S':
                    System.out.println("501");
                    break;
                case 'T':
                    System.out.println("105");
                    break;
            }
        }
    }
}